File: alternatingSums.py
def alternatingSums(a):
    return [sum([a[x] for x in range(len(a)) if x%2==0])] + [sum([a[x] for x in range(len(a)) if x%2!=0])]

def alternatingSums_V1(a):
    r=[0,0]
    i,z=0,len(a)
    while True:
        if i<z:r[0]+=a[i]        
        if i+1<z:r[1]+=a[i+1]
        if i>=z:break
        i+=2
    return r

File: alternatingSums.java
int[] alternatingSums(int[] a) {
    int[] output = new int[2];
    output[0] = 0;
    output[1] = 0;
    for(int x=0;x<a.length;x++){        
        if(x%2==0){
            output[0] += a[x];   
        } else {
            output[1] += a[x];
        }
    }
    return output;
}